### DISCUSSION TITLE
All players invisible after respawn

### DISCUSSION THREAD

**Discussion by matserix** [Minestom/Minestom#1537] [Help]
When a player dies and respawns he can't see other players, like shown in the video. 

How can I change this behaviour? I already tried by updating the viewer rule after respawn

https://user-images.githubusercontent.com/113344017/204085524-33e9e6db-8259-4252-8c6f-218446123589.mp4

**Comment by GreatWyrm** [comment]
Currently tracked as issue #1480. Will be fixed by PR #1497.
